TESTING.md

```diff
@@ -102,7 +102,7 @@ Each plugin can test various functionalities in their bwc tests and add more sce
 
 The BWC tests running on distribution level are using the same framework from OpenSearch. The test cluster is spin up with the `latest` distribution bundle of provided version exclusively when the project is initialized with property `-PcustomDistributionDownloadType=bundle`.
 
-The distribution bundle will be obtained with this format URL: https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/__REPLACE_VERSION__/latest/linux/x64/tar/dist/opensearch/opensearch-__REPLACE_VERSION__-linux-x64.tar.gz.
+The distribution bundle will be obtained with this format URL: `https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/__REPLACE_VERSION__/latest/linux/x64/tar/dist/opensearch/opensearch-__REPLACE_VERSION__-linux-x64.tar.gz`.
```

Comment:
An example would be nice.

Reply:
@saratvemulapalli Updated with the example for v1.3.2. Thanks.